site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java

```diff
@@ -226,7 +226,8 @@ private Constraint publicationSystemConstraint(String query) {
     private Constraint indicatorLibraryConstraint(String query) {
         return or(
             constraint("nationalindicatorlibrary:title").contains(query),
-            constraint("nationalindicatorlibrary:methodology").contains(query)
+            constraint("nationalindicatorlibrary:definition").contains(query),
+            constraint("common:SearchableTags").contains(query)
```

Comment:
Don't need `common:SearchableTags` as it's already been added in the publicationsystem one, really they should just be one list so we don't get duplicates like this

Reply:
Hmm. I see that - although one list covering multiple doc types (namespaces) seems ugly.

Reply:
Or split common:... out into it's own list?

Reply:
I know that the NIL stuff will be extended soon, so I'm going with 3 lists.

Reply:
👍